flags.css

```diff
@@ -1,5 +1,17 @@
 /* Indonesia */
-
+#indonesia {
+    height: 200px;
+    width: 300px;
+    border: 1px solid black
+}
+#indonesia > .topSegment {
+    height: 100px;
```

Comment:
if your outer element sizes are sized correctly, it is definitely a better idea to have your internal elements as a %